src/metabase/api/pulse.clj

```diff
@@ -178,7 +178,10 @@
   [{query :dataset_query, card-id :id}]
   (binding [qp.perms/*card-id* card-id]
     (qp/process-query-and-save-execution!
-     (assoc query :async? false)
+     (-> query
+         (assoc :async? false)
+         (assoc :middleware {:process-viz-settings? true
+                             :js-int-to-string?     false}))
```

Comment:
I suppose the preview should also take into account the viz settings stored in the DB.